packages/react-components/src/components/trace-context-component.tsx

```diff
@@ -72,6 +74,8 @@ export class TraceContextComponent extends React.Component<TraceContextProps, Tr
             traceRange = new TimeRange(experiment.start - this.props.experiment.start, experiment.end - this.props.experiment.start, this.props.experiment.start);
             viewRange = new TimeRange(experiment.start - this.props.experiment.start, experiment.end - this.props.experiment.start, this.props.experiment.start);
         }
+        ThemeService.get().onThemeChange(() => this.updateBackgroundTheme());
+        const currentThemeId = ThemeService.get().getCurrentTheme().id;
```

Comment:
If you use type instead of id then you will support all light and dark themes and not just the Theia light theme.